OpenEdXMobile/res/layout/activity_webview.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Its kinda weird but if the I change the layout to `merge` tag, the background of webview goes white even when I set the background to another color. After some research I wasn't able to find the base issue, so keeping this layout as is.